lib/mock-nexmo.js

```diff
@@ -54,7 +54,8 @@ function MockNexmo(log, config) {
 
         // HACK: Enable remote tests to see what was sent
         mailer.sendMail({
-          from: `sms.${senderId}@restmail.net`,
+          from: config.smtp.sender,
+          sender: config.smtp.sender,
```

Comment:
Although also, let's be clear that I'm making mountains out of molehills here. It really doesn't matter that much to the bigger picture if I'm honest. I think I was in pedantic bike-shedder mode when I reviewed this first time.